app/common/alerts.py

```diff
@@ -20,17 +20,28 @@ def send(email, reference):
 
 
 class AlertUser:
-    """Alert User"""
+    """Alert goes via gov notify (0) or via logs (1)"""
+
+    NOTIFY_VIA_LOGGING = settings.NOTIFY_VIA_LOGGING
+    logger.debug('Notification logging (0=disabled, 1=enabled) : {}'.format(NOTIFY_VIA_LOGGING))
     alert_method = AlertViaGovNotify()
 
     def __init__(self, alerter=None):
         if alerter is not None:
             self.alert_method = alerter
 
     def send(self, email, reference):
-        try:
-            self.alert_method.send(email, reference)
-        except BaseException as e:
-            logger.exception(e)
-        finally:
-            return 201, 'OK'
+        if self.NOTIFY_VIA_LOGGING == '1':
+            try:
+                logger.info('email details {}, {}'.format(email, reference))
+            except BaseException as e:
+                logger.exception(e)
+            finally:
+                return 201, 'OK'
+        else:
+            try:
+                self.alert_method.send(email, reference)
+            except BaseException as e:
```

Comment:
This was already here I think so probably not a change for this PR but in general you don't want to be catching such a broad class of exception. Should be more specific to what's happening here